python/pyproject.toml

```diff
@@ -1,18 +1,28 @@
-import logging
-from setuptools import find_packages, setup
-from src.aac import __version__
-from os import linesep
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
 
-README_FILE_PATH = "../README.md"
-
-try:
-    with open(README_FILE_PATH, "r") as file_handler:
-        readme_description = file_handler.read()
-except Exception as exception:
-    logging.error(f"Failed to open readme file {README_FILE_PATH} with error:{linesep}  {exception}")
-    readme_description = ""
+[project]
+name = "aac"
+version =  "0.4.18"
```

Comment:
Thought we were on 0.4.22? But should be able to pull the version in a similar way here that was being done in the `setup.py` from the `src/aac/__init__.py` file. Double check. It should be making use of the dynamic flagging to be able to do so if I remember correctly. That will allow for the version plugin to remain operating as is. Otherwise, we will need to do some rework of the plugin as well.